src/statsig/constants.ts

```diff
@@ -1,26 +1,7 @@
-import {
-  QRCodeDataType,
-  QRCodeStyle,
-  StatsigDynamicConfigs,
-  StatsigExperiments,
-  StatsigFeatureGates,
-  StatsigLayers,
-} from 'src/statsig/types'
+import { StatsigDynamicConfigs, StatsigExperiments, StatsigFeatureGates } from 'src/statsig/types'
 import networkConfig from 'src/web3/networkConfig'
 
-export const LayerParams = {
-  // TODO(ACT-659): refactor to imitate defaultExperimentParamValues (more type safe, less boilerplate)
```

Comment:
seems like we didn't want to keep this enum around in the long term anyway, we wanted to do something else. can we just delete it now?